test/fixtures/users.yml

```diff
@@ -0,0 +1,12 @@
+aaa:
```

Comment:
It is best practice not to use ids in fixtures, and instead use the name of the fixtures to build relationships.